src/Raven.Studio/typescript/viewmodels/shell/about.ts

```diff
@@ -2,12 +2,14 @@ import viewModelBase = require("viewmodels/viewModelBase");
 import shell = require("viewmodels/shell");
 import license = require("models/auth/license");
 import registration = require("viewmodels/shell/registration");
+import deactivateLicenseCommand = require("commands/licensing/deactivateLicenseCommand");
 
 class about extends viewModelBase {
 
     clientVersion = shell.clientVersion;
     serverVersion = shell.serverBuildVersion;
     licenseStatus = license.licenseStatus;
+    deactivating = ko.observable<boolean>(false);
```

Comment:
by convention in v4.0 we use variable spinners for holding spinners see:
https://github.com/ravendb/ravendb/blob/v4.0/src/Raven.Studio/typescript/viewmodels/database/indexes/indexes.ts#L38